### ISSUE TITLE
Folder item does not support register method

### ISSUE THREAD

**Issue by frague59** [ecederstrand/exchangelib#319]
Hi, 

I'm currently using exchangelib version 1.10.6, and the register method is not yet supported in this release.

When do you plan to release a new pypi version with the register feature ? 

Thanks for your great lib !

**Comment by ecederstrand** [comment]
There are test failures in Travis right now that I'd like to fix first. But you can easily install from GitHub until then:

```bash
pip install git+https://github.com/ecederstrand/exchangelib.git@6a9ec3d1358a3d65fcb8eee310d1508a15ab04de
```

**Comment by frague59** [comment]
Thanks, that what I did...

**Comment by ecederstrand** [comment]
I just released v1.10.7 which contains the new features.
